src/smooth-scroll/smooth-scroll/smooth-scroll.ts

```diff
@@ -3,10 +3,10 @@ import {ease} from "../../util/easing";
 import {disableScrollSnap, DisableScrollSnapResult} from "../../util/disable-scroll-snap";
 
 /**
- * The duration of a smooth scroll
+ * The duration of a smooth scroll in ms
  * @type {number}
  */
-const SCROLL_TIME = 15000;
+const SCROLL_TIME = 250;
```

Comment:
Just setting this to always be 250ms for now, works well for sky and is comparable to the speed of the other platforms, will raise a ticket to make this dynamic from the import so we can have more flexibility but in the interest of time leaving as is for now

Reply:
Rather than bother making it nice with imports and stuff could go low tech  `window.__SMOOTH_SCROLL_TIME__ || 250`, already patching the scroll method 🤷